w4af/core/ui/api/utils/scans.py

```diff
@@ -46,7 +59,7 @@ def create_temp_profile(scan_profile):
     scan_profile_file = os.path.join(tempdir, '%s.pw4af' % uuid4())
     with open(scan_profile_file, 'w') as profile_fh:
         profile_fh.write(scan_profile)
-
+        
```

Comment:
Looks like you're just changing whitespace formatting here. Please revert this to the original.

Reply:
It's weird because I didn't add that space. in line 62. It's not in my local code.

Reply:
:shrug: It didn't get there by itself :)

Reply:
What plugins or tools are you using for validate a format is being used ?

Reply:
I use Codium, or vim. I don't do any auto-formatting besides that.